ocr/tess/crop_morphology.py

```diff
@@ -115,20 +130,27 @@ def find_components(edges, max_components=16):
     # Perform increasingly aggressive dilation until there are just a few
     # connected components.
     count = 21
-    dilation = 5
-    n = 1
+    # dilation = 5
+    n = 15
```

Comment:
Some dilation is always desirable to merge adjacent text components. This makes a difference for images with a small amount of text. Example: 710544b